cumulus_library/studies/core/reference_sql/builder_practitioner_prereq.sql

```diff
@@ -0,0 +1,17 @@
+-- noqa: disable=all
```

Comment:
You know, it's probably worth a CI job that runs reference_sql and complains if there's any new/changed/deleted files